### PR TITLE
Created folder structure

### PR DESCRIPTION
**Pull request by mariusandreiratiu**
This solves #3

### CODE REVIEW

**Comment by mariusandreiratiu** [comment]
this solves #3

File: design/design_readme.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Here, the design documents/ideas.
```
Comment (aratiu) [inline design/design_readme.txt:1]: keep file name simple: readme.txt

File: docs/docs_readme.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Here, sources of documentation which helped in developping the app (pdfs, docs, yt videos, etc).
```
Comment (aratiu) [inline docs/docs_readme.txt:1]: keep filename simple: readme.txt

File: src/back_end/back_end_readme.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Here, the back-end applications.
```
Comment (aratiu) [inline src/back_end/back_end_readme.txt:1]: I don't think we need back_end for this app, so I suggest to replace it with tests and we will add test description.

File: src/front_end/front_end_readme.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Here, the front end applications.
```
Comment (aratiu) [inline src/front_end/front_end_readme.txt:1]: Keep simple name: readme.txt
Also rename the folder to app

**Pull request closed without merging**